modules/social_features/social_core/social_core.api.php

```diff
@@ -62,3 +62,26 @@ function hook_social_core_block_visibility_path() {
 /**
  * @} End of "denyaccesstoblock hooks".
  */
+
+/**
+ * Hooks to alter excluded CT for default title.
+ *
+ * @hidedefaultitle hooks
+ * @{
+ */
+
+/**
+ * Provide a method to alter array of CT where need to hide default title.
```

Comment:
@dmytriikaiun Please change the description to something more generic, in future it might be used not only for fixing titles.